src/cincinnati/mod.rs

```diff
@@ -130,6 +132,59 @@ impl Cincinnati {
     }
 }
 
+/// Write MOTD indicating a dead-end release, with the passed `reason`.
+/// If `reason` is `None`, the MOTD written is left as an empty file.
+fn refresh_deadend_motd(reason: Option<String>) {
+    // Avoid showing partially-written messages using tempfile and
+    // persist (rename).
+    let f = tempfile::Builder::new()
+        .prefix("deadend")
+        .suffix("motd.partial")
+        // Create the tempfile in the same directory as the final MOTD,
+        // to ensure proper SELinux labels are applied to the tempfile
+        // before renaming.
+        .tempfile_in("/run/zincati/public/motd.d");
+    let mut f = match f {
+        Ok(file) => file,
+        Err(e) => {
+            log::warn!("failed to create dead-end release MOTD tempfile: {}", e);
+            return;
+        }
+    };
+
+    if let Some(reason) = reason {
+        if let Err(e) = writeln!(
+            f,
+            "{} this release is a dead-end and won't update: {}",
+            // Fixed color 11 - a brighter shade of yellow.
+            Fixed(11).paint("Notice:"),
```

Comment:
I'd stick this to plain ASCII without fancy colors.

Reply:
Updated! Thought I'd try to make it stand out, but it is simpler to keep it plain characters.